packages/eds-core-react/src/components/TopBar/TopBar.stories.tsx

```diff
@@ -58,9 +67,15 @@ export const Introduction: Story<TopbarProps> = (props): JSX.Element => {
         </TopBar.Header>
       </TopBar>
       <BodyWrapper>
-        <p>Top of page</p>
-        <p>Middle of page</p>
-        <p>Bottom of page</p>
+        <Typography group="input" variant="text">
```

Comment:
Noticing the wrapper is gone which made the content overflow to demonstrate sticky topbar with scroll (see https://eds-storybook-react.azurewebsites.net/?path=/docs/navigation-topbar--introduction). The "top of page" etc text doesn't really make sense without it, so can be removed I guess

Reply:
Wow, i mess up big time. I took wrapper away while working on decorators. I completely forgot about `TopBar` being sticky. You saved the day @oddvernes 🙌  
I will put this wrapper back.